docs/Advanced-Usage/Conda.rst

```diff
@@ -0,0 +1,183 @@
+Non-Source Dependency Management
+================================
+
+In :doc:`/Initial-Setup/Setting-up-your-Manager-Instance`, we quickly copy-pasted the contents
+of ``scripts/machine-launch-script.sh`` into the EC2 Management Console and
+that script installed many dependencies that FireSim needs using
+`conda <https://conda.io/en/latest/index.html>`_,  a platform-agnostic package
+manager, specifically using packages from the `conda-forge community <https://conda-forge.org/#about>`_.
+
+In many situations, you may not need to know anything about conda.  By default, the
+``machine-launch-script.sh`` installs conda and all of the FireSim dependencies into ``/opt/conda``
+and adds the required setup to the system-wide ``/etc/profile.d/conda.sh`` init script to add
+``/opt/conda`` to everyone's path.
+
+However, the script is also flexible.  For example, if you do not have root access, you can specify
+an alternate install location with the ``--prefix`` option to ``machine-launch-script.sh``.  The only requirement
+is that you are able to write into the install location.  See ``machine-launch-script.sh --help`` for more details.
+
+.. warning::
+
+    Without root access you can do many things, like :doc:`/Building-a-FireSim-AFI`,
+    :ref:`meta-simulation` of a FireSim system using Verilator or even developing new features in FireSim.
+    However, to :ref:`run a simulation on a F1 FPGA <running_simulations>` , FireSim currently requires that
+    you are able to act as root via ``sudo``.
+
+Updating a Package Version
+--------------------------
+
+If you need a newer version of package and you are brave, the most expedient method to see whether there
+is a newer version available on `conda-forge`_ is to run ``conda update <package-name>``.  If you are lucky,
+and the dependencies of the package you want to update are simple, you'll see output that looks something like
+this:
+
+::
+
+    bash-4.2$ conda update moto
+    Collecting package metadata (current_repodata.json): done
+    Solving environment: done
+
+    ## Package Plan ##
+
+      environment location: /opt/conda
+
+      added / updated specs:
+        - moto
+
+
+    The following NEW packages will be INSTALLED:
+
+      graphql-core       conda-forge/noarch::graphql-core-3.2.0-pyhd8ed1ab_0
+
+    The following packages will be UPDATED:
+
+      moto                                  2.2.19-pyhd8ed1ab_0 --> 3.1.0-pyhd8ed1ab_0
+
+    Proceed ([y]/n)?
+
+
+The addition of ``graphql-core`` makes sense when looking at the `diff of moto's setup.py between
```

Comment:
```suggestion
The addition of ``graphql-core`` makes sense when looking at the diff of moto's setup.py between
```

Reply:
Need that backtick for the link to render properly I think.

Reply:
Maybe I should join line 60 back into 59 so it is easier to tell that it is a link to a specific diff on GH.